src/pages/home/index.js

```diff
@@ -1,12 +1,53 @@
+                    {image:"https://us.123rf.com/450wm/bk2kz/bk2kz1908/bk2kz190800001/129311633-motley-dahlia-flower-isolated-on-a-white-background.jpg?ver=6", name:"Dahlia", price: "£4.99", id:6},
+                    {image:"https://thumbs.dreamstime.com/b/magnolia-flowers-white-background-32217534.jpg", name: "Magnolia", price:"£3.99", id:7},
+                    {image:"https://i.pinimg.com/736x/1c/81/d4/1c81d46dcb8b85e9a8f280550ed036ec.jpg", name: "Iris", price: "£3.99", id:8},
+                    {image:"https://t4.ftcdn.net/jpg/00/90/31/97/240_F_90319705_KOFtfQLVJRWyiXyZqNgHSlmbBIUP31BE.jpg", name:"Jasmine", price:"£3.99", id:9},
+                    {image:"https://envato-shoebox-0.imgix.net/a1ed/4aa3-96c5-4b85-9e82-3541c76782b2/DSC_0689a.jpg?auto=compress%2Cformat&fit=max&mark=https%3A%2F%2Felements-assets.envato.com%2Fstatic%2Fwatermark2.png&markalign=center%2Cmiddle&markalpha=18&w=1600&s=fd796825bbfa47de64ae04087c6fff55", name: "Lavender", price:"£2.99", id:10},
+                    {image:"https://img.freepik.com/free-photo/pink-peony-isolated-white-background_14223-132.jpg?size=626&ext=jpg", name: "Peony", price:"£2.99", id:11},
+                    {image:"https://previews.123rf.com/images/dizolator/dizolator1707/dizolator170700116/82255105-bright-pink-and-white-petunia-flower-isolated-on-white-background.jpg", name:"Petunia", price: "2.99", id:12},
+                    {image:"https://previews.123rf.com/images/belchonock/belchonock1803/belchonock180360738/97828079-beautiful-freesia-flowers-on-white-background.jpg", name:"Freesia", price: "£1.99", id:13},
+                    {image:"https://as1.ftcdn.net/jpg/01/89/35/42/220_F_189354246_R3rJUguQqt7KnUoWgmjog9rpVYuppxI6.jpg", name: "Daisy Gerbera", price: "£1.99", id:14},
+                    {image:"https://media.istockphoto.com/photos/bluebell-on-white-background-picture-id488833695", name: "Bluebell", price: "£1.99", id:15}
+]
+
+const List = () => {
+  return(
+      ItemsList.map((ItemsList) => {
+          return <SaleItem image={ItemsList.image} name={ItemsList.name} price={ItemsList.price} id={ItemsList.id} />
+      })
+  )
+}
+
 
-const Home = () => {
   return(
-      <div>
-        <p>This is the start of you first real react project.</p>
-        <p>I've done some of the basic setup for you, like adding react-router for routing.</p>
-        <p>You should have a trello board with all the tasks you need to complete on it. Along with a design file from figma that you should work from.</p>
-        <p>This is going to be a much larger chalenge than any of your previous work, so make sure that you plan each part out and refer to notes!</p>
+      <div className="container">
+        <h3 className="title">Welcome to the store!</h3>
+        <button className="button" onClick={handleClick}>Check Out</button>
+      <br />
+      <br />
+        <div className="box" onClick={itemSelect}>
```

Comment:
The onClick should be passed to `SaleItem` as a prop, right now I don't think this will work